GitUI/CommandsDialogs/FormPush.cs

```diff
@@ -773,6 +769,11 @@ private void BranchSelectedValueChanged(object sender, EventArgs e)
 
             if (_NO_TRANSLATE_Branch.Text != HeadText)
             {
+                if (!RemoteBranch.Items.Contains(_NO_TRANSLATE_Branch.Text))
+                {
+                    RemoteBranch.Items.Add(_NO_TRANSLATE_Branch.Text);
```

Comment:
A remote name is not always the same as a local name. This adds an incorrect remote for me.
The local as remote is added incorrectly for me.
Not a new problem, also added in UpdateRemoteBranchDropDown()

![image](https://github.com/gitextensions/gitextensions/assets/6248932/3af7f545-5c29-477c-855e-9adf6834b673)

Reply:
Fixed: adding only if no tracking branch found.